### DISCUSSION TITLE
Getting League Standings

### DISCUSSION THREAD

**Discussion by dylan-ascent** [cwendt94/espn-api#448] [Ideas] [answered]
Is there a way for me to use the repo to get my FF leagues standings so I can put together automated email news letter around that? If not that would be pretty cool.

**Comment by mpolowski** [comment, marked as answer]
Yes you should be able to pull them using one of the helper functions found in the wiki here: https://github.com/cwendt94/espn-api/wiki/League-Class

```python

def standings() -> List[Team]:


#other helper functions
def top_scorer() -> Team:
def least_scorer() -> Team:
def most_points_against() -> Team:
def top_scored_week() -> Tuple(Team, int):
def least_scored_week() -> Tuple(Team, int):
def get_team_data(team_id: int) -> Team:
```
